CONTRIBUTING.md

```diff
@@ -23,21 +23,21 @@ The issues page on GitHub is for tracking bugs and feature requests. When posing
 
 If you like to contribute with a bug fix or a new feature, start by posting an issue and discussing the best way to implement it.
 
-Unlike most projects, development for this repository is carried out on the `2.0.0` branch. This is because the `master` branch contains the latest stable release of the project.
-The code in `2.0.0` is merged to the `master` branch after a final review and a round of testing.
+Unlike most projects, development for this repository is carried out on the `3.0.0` branch. This is because the `master` branch contains the latest stable release of the project.
```

Comment:
@AmilaSamith this has to be `3.0.x` if I am not mistaken.

Reply:
Yes it is I will make the change